src/components/tasks/ManageVariants.vue

```diff
@@ -0,0 +1,687 @@
+                    placeholder="Field Type"
+                    class="flex-grow-1"
+                  />
+
+                  <PvInputText
+                    v-if="field.type === 'string'"
+                    v-model="field.value"
+                    placeholder="Field Value"
+                    class="flex-grow-1"
+                  />
+                  <PvInputNumber
+                    v-if="field.type === 'number'"
+                    v-model="field.value"
+                    placeholder="Field Value"
+                    class="flex-grow-1"
+                  />
+                  <PvDropdown
+                    v-if="field.type === 'boolean'"
+                    v-model="field.value"
+                    placeholder="Field Value"
+                    :options="booleanDropDownOptions"
+                    option-label="label"
+                    option-value="value"
+                    class="flex-grow-1"
+                  />
+                  <PvButton type="button" class="w-2" @click="removeField(field.name, newParams)">Delete</PvButton>
+                </div>
+              </div>
+            </div>
+            <PvButton label="Add Param" text icon="pi pi-plus" class="w-2 m-auto my-4" @click="newParam" />
+          </div>
+          <div class="flex flex-row align-items-center justify-content-center gap-2 flex-order-0 my-3">
+            <div class="flex flex-row align-items-center">
+              <PvCheckbox
+                v-model="variantCheckboxData"
+                input-id="chbx-demoVariant"
+                name="variantCheckboxData"
+                value="isDemoVariant"
+              />
+              <label class="ml-1 mr-3" for="chbx-demoVariant">Mark as <b>Demo Variant</b></label>
+            </div>
+            <div class="flex flex-row align-items-center">
+              <PvCheckbox
+                v-model="variantCheckboxData"
+                input-id="chbx-testVariant"
+                name="variantCheckboxData"
+                value="isTestVariant"
+              />
+              <label class="ml-1 mr-3" for="chbx-testVariant">Mark as <b>Test Variant</b></label>
+            </div>
+            <div class="flex flex-row align-items-center">
+              <div class="flex flex-row align-items-center">
+                <PvCheckbox
+                  v-model="variantCheckboxData"
+                  input-id="chbx-registeredVariant"
+                  name="variantCheckboxData"
+                  value="isRegisteredVariant"
+                />
+                <label class="ml-1 mr-3" for="chbx-externalVariant">Mark as <b>Registered Variant</b></label>
+              </div>
+            </div>
+          </div>
+          <div class="form-submit">
+            <PvButton type="submit" label="Submit" class="submit-button" severity="primary" />
+          </div>
+        </form>
+      </div>
+    </PvTabPanel>
+
+    <PvTabPanel header="Update Variant">
```

Comment:
I feel that we could probably find a better method to display the switch between `update/create` for variants and tasks than nested panels. I know that you had mentioned that you were planning on switching. Could I suggest the use of [radio buttons ](https://primevue.org/radiobutton/) to toggle between the two, similar to how @Emily-ejag uses them to toggle between the different modes of consent picking?

Reply:
I agree that it isn't ideal, but I think that I will hold off on any additional changes until we receive some feedback from users. For now, I made sure that the nested tabs are entirely navigable by keyboard actions.